client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -186,8 +206,12 @@ function PaymentMethodAdd(): ReactElement {
 							<div className="payment-method-add__navigation-buttons">
 								<Button
 									className="payment-method-add__back-button"
-									href="/partner-portal/payment-methods/"
-									disabled={ isStripeLoading }
+									href={
+										product
+											? addQueryArgs( { product }, partnerPortalBasePath( '/issue-license' ) )
```

Comment:
Props for retaining some benefits from the last-minute implementation we had before so we could pre-select a product for the customer when they go back!